tests/functional/amo_sign_up.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/fx-desktop'
+], function (intern, registerSuite, FunctionalHelpers, FxDesktopHelpers) {
+
+  var MIGRATE_PARAMS = 'state=state&migration=amo&scope=scope';
+  var listenForFxaCommands = FxDesktopHelpers.listenForFxaCommands;
```

Comment:
`listenForFxaCommands` is only needed when `context=fx_desktop_v1` and can be removed here.